great_expectations/core/util.py

```diff
@@ -75,8 +75,8 @@
 from great_expectations.optional_imports import SQLALCHEMY_NOT_IMPORTED, sqlalchemy
 
 try:
-    LegacyRow = sqlalchemy.engine.row.LegacyRow
-    TextClause = sqlalchemy.sql.elements.TextClause
+    LegacyRow = sqlalchemy.engine.row
+    TextClause = sqlalchemy.sql.elements
 except ImportError:
```

Comment:
```suggestion
    LegacyRow = sqlalchemy.engine.row.LegacyRow
    TextClause = sqlalchemy.sql.elements.TextClause
except (ImportError, AttributeError):
```
Should be this?

Reply:
If sqlalchemy has not been imported successfully it is actually a `NotImported` object so `sqlalchemy.engine` will throw a `ModuleNotFound` error. `NotImported` should hide this `AttributeError` exception.